osbuild/pipeline.py

```diff
@@ -98,7 +98,7 @@ def run(self,
                                       sources_output):
                 r = build_root.run(
                     [f"/run/osbuild/lib/stages/{self.name}"],
-                    binds=[f"{tree}:/run/osbuild/tree"],
```

Comment:
Should `tree` have a type annotation in its declaration, or alternatively, should we add a test to verify that this works in the non-string case?